level-2/store.html

```diff
@@ -15,26 +15,32 @@
     <header>
         <img id="logo" src="/img/karma-logo.svg" width="30px" alt="karma logo" />
         <nav>
-            <ul class="navigation_list">
-                <li class="navigation_item"> 
-                    <a class="navigation__link" href="/index.html">Meet Karma</a>
-                </li>
-                <li class="navigation_item">
-                    <a class="navigation__link" href="#">How it works </a>
-                </li>
-                <li class="navigation_item">
-                    <a class="navigation__link" href="/level-2/store.html">Store </a>
-                </li>
-                <li class="navigation_item">
-                    <a class="navigation__link" href="#">Blog </a>
-                </li>
-                <li class="navigation_item">
-                    <a class="navigation__link" href="#">Help </a>
-                </li>
-                <li class="navigation_item">
-                    <a class="navigation__link" href="#">Login </a>
-                </li>
-            </ul>
+            <div id="menuToggle">
+                <input id="ham" name="ham" type="checkbox"/>
+                <span></span>
+                <span></span>
+                <span></span>
+                <ul class="navigation_list">
+                    <li class="navigation_item"> 
+                        <a class="navigation__link" href="/index.html">Meet Karma</a>
+                    </li>
+                    <li class="navigation_item">
+                        <a class="navigation__link" href="#">How it works </a>
+                    </li>
+                    <li class="navigation_item">
+                        <a class="navigation__link" href="/level-2/store.html">Store </a>
+                    </li>
+                    <li class="navigation_item">
+                        <a class="navigation__link" href="#">Blog </a>
+                    </li>
+                    <li class="navigation_item">
+                        <a class="navigation__link" href="#">Help </a>
+                    </li>
+                    <li class="navigation_item">
+                        <a class="navigation__link" href="#">Login </a>
+                    </li>
+                </ul>
+            </div>
```

Comment:
Also, I really like you using the "required" parameter in a form, it's really important